### ISSUE TITLE
Are there any benchmarks?

### ISSUE THREAD

**Issue by sancarn** [NeuraLegion/shainet#86]
Hi all,

I've looked around a bit and can't find any benchmarks of shainet with deep learning frameworks torch, tensorflow, etc..

I know benchmarks are generally not that important, but out of curiosity, are any publicly available?

**Comment by ArtLinkov** [comment]
Hi @sancarn, thanks for taking interest in our project.
We did not create any benchmarks comparing SHAInet to other frameworks as we feel it is not mature enough for that yet.
Since it is a free-time project we are currently focusing on implementing all the ideas that hopefully will set this framework apart from the others (i.e special neurons).
If you feel like adding a benchmark is something you would want to do, we would be grateful and will happily accept a PR. Please keep in mind though that currently we take a considerably longer time to train since we do not use matrices in SHAInet (yet). Our hope is that the end result of the trained models will justify the longer wait :) However it means there is still some trial and error ahead to see how to best take advantage of our ideas with some interesting models
